### DISCUSSION TITLE
Where does the "scene_add" code go?

### DISCUSSION THREAD

**Discussion by donutVillan** [Koenkk/zigbee2mqtt#16074] [General]
Where do I actually put the "scene_add" code ([Z2M - scenes](https://www.zigbee2mqtt.io/guide/usage/scenes.html))?

**Comment by donutVillan** [comment]
One week and no answer??! I expected this to be the easiest question ever...

**Reply by malaakso** [reply]
Quoting that link:

"sending to zigbee2mqtt/[GROUP_OR_DEVICE_FRIENDLY_NAME]/set"
